paasta_tools/utils.py

```diff
@@ -2098,7 +2098,7 @@ def get_slack_token(self) -> str:
         return self.config_dict.get('slack', {}).get('token', None)
 
     def get_tron_config(self) -> dict:
-        return self.config_dict.get('tron', {})
+        return self.config_dict.get('tron_api_endpoints', {})
```

Comment:
Are you going to make a review in puppet to deploy this new config file?

You are going to have to provide both old and new

Reply:
Is the old one used anywhere? I didn't find tron.json in /etc/paasta.